### ISSUE TITLE
Add a grid search strategy for ARIMA modeling

### ISSUE THREAD

**Issue by samarthbhargav** [egovernments/analytics#52]
Right now, the ARIMA notebook requires manual selection of the (p, d, q) parameters. It should be automated, in the following way:
- Build a grid of possible (p, d, q) params
- Split the data into a train, test, and validation set
- Select [1] one param set, fit a model, compute accuracy[2]. 
- Repeat until desired. 
- Pick the param set and model which scores highest on the validation set
- Compute final accuracy on the test set

[1]:
The sampling can be done in two ways, and both needs to be explored. This entirely depends on the size of the sample space.
1. Iterate through the entire space: Possible only if it is small, and if modeling is fast
2. Perform Random Sampling of the space: Might not give us the best model

[2]:
As of today, the accuracy metric is still not finalized. Refer #39
